custom/enikshay/user_setup.py

```diff
@@ -267,50 +269,104 @@ def __init__(self, *args, **kwargs):
         super(ENikshayNewMobileWorkerForm, self).__init__(*args, **kwargs)
 
 
-class ENikshayUserDataEditor(CustomDataEditor):
-    """Custom User Data (everywhere it turns up)"""
-    def __init__(self, *args, **kwargs):
-        super(ENikshayUserDataEditor, self).__init__(*args, **kwargs)
+# pcp -> MBBS
+# pac -> AYUSH/other
+# plc -> Private Lab
+# pcc -> pharmacy / chemist
+# dto -> Field officer??
```

Comment:
Nice, thanks - [the spec](https://docs.google.com/spreadsheets/d/1R1rv3bpJxPRQSeTWDMU3IXX4lVhWMfZvLnxye3riizQ/edit#gid=1458284637) said dto, but seemed unsure